From 1c0fcc2a63c9e9f4731020b1f235be73c25bc1cc Mon Sep 17 00:00:00 2001
From: chao zeng <chao.zeng@siemens.com>
Date: Wed, 26 Apr 2023 15:44:46 +0800
Subject: [PATCH 120/120] dts: disable the 1.8v for the sd card

Some of sd cards can not work when the mpcie inserted,
currently don't find out the root cause.

This is a workaround to disable the voltage
adjust of the sd card make the card work

Signed-off-by: chao zeng <chao.zeng@siemens.com>
---
 arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
index 733d6925c613..013c62202e7d 100644
--- a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
@@ -664,6 +664,7 @@ &sdhci1 {
 	pinctrl-0 = <&main_mmc1_pins_default>;
 	ti,driver-strength-ohm = <50>;
 	disable-wp;
+	no-1-8-v;
 };
 
 &usb0 {
-- 
2.34.1

